pkg/types/validation/installconfig.go

```diff
@@ -324,8 +324,12 @@ func validateNetworking(n *types.Networking, singleNodeOpenShift bool, fldPath *
 		allErrs = append(allErrs, field.Required(fldPath.Child("networkType"), "network provider type required"))
 	}
 
-	if singleNodeOpenShift && n.NetworkType == string(operv1.NetworkTypeOpenShiftSDN) {
-		allErrs = append(allErrs, field.Invalid(fldPath.Child("networkType"), n.NetworkType, "networkType OpenShiftSDN is currently not supported on Single Node OpenShift"))
+	if n.NetworkType == string(operv1.NetworkTypeOpenShiftSDN) {
+		if singleNodeOpenShift {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("networkType"), n.NetworkType, "networkType OpenShiftSDN is currently not supported on Single Node OpenShift"))
+		} else {
+			logrus.Warnf("networkType OpenShiftSDN is going to be deprecated for new clusters in 4.15, please consider using OVNKubernetes")
```

Comment:
I think we're running into the ["definition of deprecated" issue](https://github.com/openshift/installer/pull/7720/files#r1481575895) here again.
At the time this PR was created, the error in 4.15 said "networkType OpenShiftSDN is deprecated, please use OVNKubernetes", so this was consistent with that, but not the common meaning of 'deprecated' (which is "it still works but don't start using it because it's going to stop working in a later version").

The error message has since been [changed](https://github.com/openshift/installer/pull/8092) to "networkType OpenShiftSDN is not supported, please use OVNKubernetes".

So I think this should read something like:

> networkType OpenShiftSDN is deprecated, please consider using OVNKubernetes